magenta-lib/src/main/scala/magenta/tasks/UpdateCloudFormationTask.scala

```diff
@@ -27,7 +28,12 @@ case class UpdateCloudFormationTask(cloudFormationStackName: String, template: P
     MessageBroker.info(s"Parameters: $actualParameters")
 
     if (CloudFormation.describeStack(cloudFormationStackName).isDefined)
-      CloudFormation.updateStack(cloudFormationStackName, template.string, actualParameters)
+      try {
+        CloudFormation.updateStack(cloudFormationStackName, template.string, actualParameters)
+      } catch {
+        case ase:AmazonServiceException if ase.getMessage contains "No updates are to be performed." =>
```

Comment:
I think that the error code returns a pretty generic value from the failure examples in Riff-Raff in CODE. e.g. http://riffraff.code.dev-gutools.co.uk/deployment/view/edf42137-200b-4021-8925-650881a79255
Doesn't seem to be a specific identifiable code that is likely to be unique.